Add Sponsored Site page. (#1311)
platformsh/platformsh-docs: src/overview/pricing/sponsored.md

# Sponsored Community Sites

Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.

## Eligibility

* A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)
* A community-oriented conference or similar event run either by the community or a related non-profit corporation.  For-profit events and organizations are not eligible.
* An offline IT community organization, such as a local user group for a Free and Open Source project.

> * An online non-profit community project that supports one of the above.

Reviewer: @Crell 
Can we include some social project such as https://glocal.org.br/aceleradora/

# Sponsored Community Sites

Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.

## Eligibility

> * A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)

Reviewer 2: ```suggestion
* A non-trivial software project released under a Free and Open Source license (GPL, BSD, MIT, Apache, etc.)
```

Author: The * here is to indicate AGPL, LGPL, GPL, any version, etc.

* A community-oriented conference or similar event run either by the community or a related non-profit corporation.  For-profit events and organizations are not eligible.
* An offline IT community organization, such as a local user group for a Free and Open Source project.
* An online non-profit community project that supports one of the above.

The Platform.sh Developer Relations team manages the program and collectively has final say on what projects are eligible for sponsorship.

> Platform.sh will periodically reevaluate sponsored sites to ensure they remain eligible, and may with a month notice terminate the free service at its own will. Platform.sh may terminate this policy providing a month notice to allow users to migrate away

Reviewer 2: ```suggestion
Platform.sh will periodically reevaluate sponsored sites to ensure they remain eligible, and may terminate the free service at its own will. Platform.sh will provide a month notice to allow users to migrate in these cases.
```

Platform.sh may update from time to time the widget’s contents and the sponsored site will update the contents in a timely fashion.

The sponsored site may ask, and Platform.sh shall accept variations of the widget as long as these fall within the branding guidelines of Platform.sh.

Platform.sh may announce on its own media presence both online and offline that it hosts the said project, and kindly ask the sponsored site communicate at least once over its usual social media presence (Twitter, Mailing list, etc) the sponsored hosting.

If the eligible project is a registered non-profit organization in the US or Germany, it is also expected to provide paperwork to register an in-kind donation to the organization of the equivalent price of the service ($50 USD/month or the EUR equivalent), for tax purposes.

Sponsored projects will receive the same level of support as any other self-service hosted customer site.

> ## Contact

Reviewer 2: missing text?

Author: Waiting on the form from Chris Yates so we can embed a button for it.
